ui/packages/atlasmap/src/UI/ColumnMapper/Column.tsx

```diff
@@ -13,11 +13,13 @@ const styles = StyleSheet.create({
 });
 
 export interface IColumnProps {
+  id: string;
```

Comment:
There is no need for this, `Column` already accepts any `data-` attribute you pass.
We just need to set the right `data-testid` where `Column` is used